salsah/src/typescript_interfaces/basicMessageComponents.ts

```diff
@@ -122,11 +122,6 @@ export module basicMessageComponents {
          * Knora status code
          */
         status:KnoraStatusCode;
-
-        /**
-         * The current user's data
-         */
-        userdata:userdata;
```

Comment:
Now I've moved it to `sessionResponseFormats.ts`, which has the whole login response format. There's a sample in `sampleLoginResponse.ts`.